http_client/fetch_ic/call.ts

```diff
@@ -126,7 +126,7 @@ async function prepareRequestBody(
         init.body instanceof Float32Array ||
         init.body instanceof Float64Array
     ) {
-        return new Uint8Array(init.body);
+        return new Uint8Array(init.body as ArrayBuffer);
```

Comment:
Let's talk about this, if you just get rid of BigUint64Array and BigInt64Array in the union the error gets fixed. We should consider just doing that, and maybe this is some bug we should open an issue for somewhere.

Reply:
Maybe we should still file that issues somewhere?